third_party/android_deps/BUILD.gn

```diff
@@ -7,6 +7,8 @@ group("android_deps") {
     ":com_android_installreferrer_java",
     ":com_brave_playlist_java",
     ":com_github_bumptech_glide_java",
+    ":com_google_android_datatransport_transport_backend_cct_java",
```

Comment:
`android_deps` target does not appear to actually be used anywhere so can we just remove it?